Simplify readme (#192)
pyansys/pymechanical: doc/source/contributing.rst

 
The following contribution information is specific to PyMechanical.

Install in developer mode
-------------------------

Installing PyMechanical in developer mode allows
you to modify the source and enhance it.

.. note::

    Before contributing to the project, ensure that you are thoroughly familiar
    with the `PyAnsys Developer's Guide`_.

To install PyMechanical in developer mode, perform these steps:

#. Clone the ``pymechanical`` repository:

   .. code:: bash

      git clone https://github.com/pyansys/pymechanical

#. Access the ``pymechanical`` directory where the repository has been cloned:

   .. code:: bash

      cd pymechanical

#. Create a clean Python virtual environment and activate it:

>    .. code:: bash
> 
>       # Create a virtual environment
>       python -m venv .venv
> 
>       # Activate it in a POSIX system
>       source .venv/bin/activate
> 
>       # Activate it in Windows CMD environment
>       .venv\Scripts\activate.bat
> 
>       # Activate it in Windows Powershell
>       .venv\Scripts\Activate.ps1

Reviewer: ```suggestion
.. tab-set::

    .. tab-item:: Windows

        .. tab-set::

            .. tab-item:: CMD

                .. code-block:: text

                    python -m venv .venv
                    .venv\Scripts\activate.bat

            .. tab-item:: PowerShell

                .. code-block:: text

                    python -m venv .venv
                    .venv\Scripts\Activate.ps1

    .. tab-item:: Linux/UNIX

        .. code-block:: text
            python -m venv .venv
            source .venv/bin/activate
  
```

method.

Documentation
-------------

For building documentation, you can run the usual rules provided in the
`Sphinx`_ ``make`` file. Here is an example:

.. code:: bash

    #  build and view the doc from the POSIX system
    make -C doc/ html && your_browser_name doc/html/index.html

>     # build and view the doc from CMD / PowerShell environment

Reviewer: ```suggestion
    # build and view the doc from Windows environment
```

    .\doc\make.bat clean
    .\doc\make.bat html
    start .\doc\_build\html\index.html

However, the recommended way of checking documentation integrity is to use
``tox``:

.. code:: bash

>     tox -e doc && your_browser_name .tox/doc_out/index.html

Reviewer: ```suggestion
    tox -e doc && <your_browser_name> .tox/doc_out/index.html
```
